index.html

```diff
@@ -3752,6 +3752,69 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section>
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+When using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation.
```

Comment:
could we link to the section that describes this?

Reply:
Fixed in 91cb4e6bcc438de69c8d7cebb9e9f8cc96f8276c.